src/components/timeout-dialog/validate-input.js

```diff
@@ -10,4 +10,6 @@ ValidateInput.int = (stringToValidate) => {
 
 ValidateInput.string = (stringToValidate) => stringToValidate && ((`${stringToValidate}`) || undefined);
```

Comment:
total tangent, but I'm really curious about this `(`${stringToValidate}`) || undefined`